petri/src/vm/hyperv/powershell.rs

```diff
@@ -0,0 +1,410 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//! Functions for creating Hyper-V VMs.
```

Comment:
this seems like another module that might be worth sharing with things like `hypestv`, in some shared "Rust <-> Hyper-V management" crate.